src/main/java/org/mozilla/iot/webthing/errors/PropertyError.java

```diff
@@ -0,0 +1,11 @@
+package org.mozilla.iot.webthing.errors;
+
+public class PropertyError extends Exception {
+    public PropertyError() {
+        super("");
```

Comment:
Maybe give it a more descriptive default message?

Reply:
Done.